integration-tests/applications/src/test/java/com/xsk/integration/tests/applications/kyma/KymaResponseBody.java

```diff
@@ -0,0 +1,13 @@
+package com.xsk.integration.tests.applications.kyma;
+
+public class KymaResponseBody {
+    private String access_token;
```

Comment:
This could be `final` and it's bad practice to use snake_case in Java. Rename it to `accessToken` and change the getter function as well